src/librustdoc/html/static/main.js

```diff
@@ -113,6 +113,7 @@ function defocusSearchBar() {
     var mouseMovedAfterSearch = true;
 
     var titleBeforeSearch = document.title;
+    var searchTitle = null;
```

Comment:
Do we really need another variable? Why not just directly modify `document.title`?

Reply:
We do, but it's simpler in case we come back to the search instead of regenerate it once again.